### DISCUSSION TITLE
Not able to fetch redirect on any Google News links

### DISCUSSION THREAD

**Discussion by onegigbyte** [node-fetch/node-fetch#1754] [General]
Hi, I was testing the `node-fetch` API on Google News since all links on it are redirects.
However, API was not able to retrieve the destination url, it pretty much gives me the same URL I entered, which is different from what I tried in browser.

For example, 

```js
// url can be any link on google.com/news
const url = 'https://news.google.com/rss/articles/CBMiUmh0dHBzOi8vd3d3LmNubi5jb20vMjAyMy8wNi8xMi91cy9waGlsYWRlbHBoaWEtaTk1LWNvbGxhcHNlLWZpcmUtbW9uZGF5L2luZGV4Lmh0bWzSAVZodHRwczovL2FtcC5jbm4uY29tL2Nubi8yMDIzLzA2LzEyL3VzL3BoaWxhZGVscGhpYS1pOTUtY29sbGFwc2UtZmlyZS1tb25kYXkvaW5kZXguaHRtbA?oc=5';

const response = await fetch(url, {
     redirect: 'follow',
     method: 'GET',
     follow: 20,
});

console.log(response.url);
// Pretty much the same as the "url", which is different from what browser gives.
```

Questions:
1. Why the destination url can not be retrieved?
2. What is Google News doing to prevent the URL tracing? Is it even a "redirect"?

**Comment by audas** [comment]
Did you ever resolve this? 
I have tried everything - follow-redirects etc and thing will work. Its so weird.
